### PR TITLE
Add moderation section to the glossary

### PR DESCRIPTION
**Pull request by Mm2PL**

### CODE REVIEW

File: docs/Glossary.md [hunk 1]
```diff
@@ -10,3 +12,14 @@ Explain common words used in Chatterino.
...
 |Split Menu|Settings or actions for the specific split|
 |Tab|Can contain multiple splits|
+
+## Moderation
+
+![moderation](images/glossary/moderation.png)
+
+|Term|Description|
+|-|-|
+|Moderation buttons|Allow you to quickly delete messages, timeout users or execute custom commands. They can be configured in the settings|
```
Comment (zneix) [inline docs/Glossary.md:22]:
```suggestion
|Moderation buttons|Allow you to quickly delete messages, timeout users or execute custom commands. They can be configured in Settings > Moderation > Moderation buttons|
```

File: docs/Glossary.md [hunk 2]
```diff
@@ -10,3 +12,14 @@ Explain common words used in Chatterino.
...
+|Moderation mode button|Toggles whether Moderation buttons are shown|
+|Usercard|Contains useful information about a user, action buttons as well as their recent messages|
+|User Timeout Buttons|Allow you time a user out from their Usercard|
```
Comment (zneix) [inline docs/Glossary.md:25]:
```suggestion
|User Timeout Buttons|Allow you time a user out from their Usercard. They can be configured in Settings > Moderation > User Timeout Buttons|
```

File: docs/Glossary.md [hunk 3]
```diff
@@ -10,3 +12,14 @@ Explain common words used in Chatterino.
...
+
+## Moderation
+
+![moderation](images/glossary/moderation.png)
+
+|Term|Description|
+|-|-|
+|Moderation buttons|Allow you to quickly delete messages, timeout users or execute custom commands. They can be configured in the settings|
+|Moderation mode button|Toggles whether Moderation buttons are shown|
+|Usercard|Contains useful information about a user, action buttons as well as their recent messages|
```
Comment (zneix) [inline docs/Glossary.md:24]:
```suggestion
|Usercard|Contains useful information about a user, action buttons as well as their recent messages. Can be opened by clicking someone's username in chat or by typing `/usercard username`|
```

**Review by zneix — CHANGES_REQUESTED** [review]
Other than few more verbose explanations I pointed out, it Looks Good To Me :+1:

**Review by zneix — APPROVED** [review]

**Review by pajlada — APPROVED** [review]

**Pull request merged**